subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsAction.kt

```diff
@@ -40,6 +40,7 @@ class InstrumentationTestsAction(
 
     override fun run() {
         logger.debug("Starting instrumentation tests action for configuration: ${params.instrumentationConfiguration}")
+        logger.info("impact analysis = ${params.impactAnalysisResult}")
```

Comment:
debug
Maybe better write this to the file system

Reply:
debug doing exactly that; done